frontends/web/src/routes/router.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { FunctionComponent } from "react";
```

Comment:
I extracted the routes from main, and made a Router Component because it made easier to maintain the routes.